data/hai/hai outfits.txt

```diff
@@ -108,6 +134,66 @@ effect "ion impact"
 	"velocity scale" -.05
 
 
+outfit "Ionic Blaster Prototype"
+	category "Guns"
+	cost 530000
+	thumbnail "outfit/hai ionic blaster"
+	"mass" 19
+	"outfit space" -19
+	"weapon capacity" -19
+	"gun ports" -1
+	weapon
+		sprite "projectile/ion blast"
+			"frame rate" 5
+		sound "ion blast"
+		"hit effect" "ion impact"
+		"inaccuracy" 4
+		"velocity" 18
+		"lifetime" 30
+		"reload" 30
+		"firing energy" 72
+		"firing force" 12
+		"firing heat" 108
+		"hit force" 72
+		"shield damage" 70
+		"hull damage" 30
+		"ion damage" 1
+	description "Since the beginning of their conflict with the rest of the Hai, the Unfettered have attempted to gain a technological edge despite their limited resources. To that end, they began developing the Ionic Blaster. Much smaller than the venerable Ion Cannon but still able to apply the same weapon jamming and energy draining effects, albeit to a much lesser extent, it also has enough raw damage to be useful in all situations."
+	description "	The design goal for the Ionic Blaster was to produce a smaller ion weapon that can can fill all the gun ports on Shield Beetles. The result is also small enough to be equipped on the Unfettered's new fighters."
+	description "	This design cannot yet be mass produced, but those that have been produced are prized by the few lucky enough to possess them."
+
+outfit "Ionic Turret Prototype"
+	category "Turrets"
+	cost 1260000
+	thumbnail "outfit/hai ionic turret"
+	"mass" 58
+	"outfit space" -58
+	"weapon capacity" -58
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/ion bolt"
+			"frame rate" 5
+		"hardpoint sprite" "hardpoint/hai ionic turret"
+		"hardpoint offset" 16.
+		sound "ion blast"
+		"hit effect" "ion impact" 5
+		"inaccuracy" 2
+		"turret turn" 2.4
+		"velocity" 12
+		"lifetime" 70
+		"reload" 70
+		"firing energy" 371
+		"firing force" 8
+		"firing heat" 259
+		"hit force" 60
+		"shield damage" 100
+		"hull damage" 50
+		"ion damage" 14
+	description "This weapon improves upon the ionization capabilities of the Ion Cannon, while also being turret-mounted, enabling it to fire at enemies in any direction, not just those directly ahead."
+	description "	This design cannot yet be mass produced, but those that have been produced are prized by the few lucky enough to possess them."
```

Comment:
I'm assuming the intent is that a shield beetle can fill its weapon space nearly perfectly with Ionic blasters, and some combination of ionic turrets + AM. Can you confirm?

Also is display name a thing yet?
If so these should just be "Ionic <thing>" and have the "Prototype" added as a display name so that you can just remove that later when/if they go into mass production.

Reply:
It isnt a thing yet and I'll be making new outfits after these that will be the non prototype version; they wont have the same stats and the player will have a say for said stats

Reply:
Yeah a shield beetle can fit 8 of these + 2 turrets + either 2 AM or another turret